includes/class-wc-payments-stripe-link-button-handler.php

```diff
@@ -55,7 +55,6 @@ public function init() {
 		}
 
 		add_action( 'woocommerce_checkout_before_customer_details', [ $this, 'display_button_html' ], -2 );
-		add_action( 'woocommerce_checkout_before_customer_details', [ $this, 'display_button_separator_html' ], -1 );
```

Comment:
Similar to the previous comment - I think we don't need the `display_button_separator_html` method in this class anymore and can safely remove it. 

The links to the method I provide here are somehow broken, so to be explicit about the method I have in mind -`WC_Payments_Stripe_Link_Button_Handler::display_button_separator_html`

Reply:
Yes indeed!